pages/index.tsx

```diff
@@ -20,9 +20,10 @@ const message =
 const sites: Site[] = [
   { name: 'Greece', url: 'https://greece.refugee.info/en-us' },
   { name: 'Italy', url: 'https://italy.refugee.info/hc/en-us' },
-  // To add Bulgaria and Hungry instances 
+  { name: 'Hungary', url: 'https://hungary.refugee.info/en-us'},
```

Comment:
the url should be https://hungary.refugee.info so the user can select a language, if we already put /en-us we won't show the user the language selector page.

You should make this change in the other urls as wel

Reply:
updated the links